devtools/server/actors/replay/connection.js

```diff
@@ -1189,6 +1195,48 @@ async function uploadAllSourcemapAssets({
   ]);
 }
 
+async function writeAllSourcemapAssets({
+  recordingId,
+  sourceMapURL,
+  sourceMapBaseURL,
+  targetContentHash,
+  targetURLHash,
+  targetMapURLHash,
+}, recording) {
+  const contentPrincipal = recording.browser.contentPrincipal;
+  const result = await fetchText(contentPrincipal, recordingId, sourceMapURL);
+  if (!result) {
+    return;
+  }
+  const mapText = result.text;
+  const id = String(Math.floor(Math.random() * 10000000000));
```

Comment:
I generate the `id` here instead of in the driver because I need it to associate the original sources to this sourcemap.